pyomo/contrib/fme/fourier_motzkin_elimination.py

```diff
@@ -68,6 +98,20 @@ class Fourier_Motzkin_Elimination_Transformation(Transformation):
         Note that these variables must all be continuous and the model must be 
         linear."""
     ))
+    CONFIG.declare('constraint_filtering_callback', ConfigValue(
+        default=_check_var_bounds_filter,
+        domain=constraint_filtering_function,
+        description="Specifies whether or not and how the transformation should"
+        " filter out trivial constraints during the transformation.",
```

Comment:
"a callback that determines whether or not new FME constraints are added to the model."